crates/core/src/view/mod.rs

```diff
@@ -97,14 +98,18 @@ pub type Bus = VecDeque<Event>;
 pub type Hub = Sender<Event>;
 
 pub trait View: Downcast + DynClone {
-    fn handle_event(&mut self, evt: &Event, hub: &Hub, bus: &mut Bus, rq: &mut RenderQueue, context: &mut Context) -> bool;
-    fn render(&self, fb: &mut dyn Framebuffer, rect: Rectangle, fonts: &mut Fonts);
     fn rect(&self) -> &Rectangle;
     fn rect_mut(&mut self) -> &mut Rectangle;
     fn children(&self) -> &Vec<Box<dyn View>>;
     fn children_mut(&mut self) -> &mut Vec<Box<dyn View>>;
     fn id(&self) -> Id;
 
+    #[allow(unused_variables)]
+    fn render(&self, fb: &mut dyn Framebuffer, rect: Rectangle, fonts: &mut Fonts) {}
+
+    #[allow(unused_variables)]
+    fn handle_event(&mut self, evt: &Event, hub: &Hub, bus: &mut Bus, rq: &mut RenderQueue, context: &mut Context) -> bool { false }
+
```

Comment:
Lots of components are just simple wrappers around their children.  Adding these default methods to the View attribute allows us to use the View defaults, and not re-write these empty methods all the time